main.go

```diff
@@ -98,7 +101,12 @@ func main() {
 		templates.SetProxiesURL(*proxiesURL)
 	}
 
-	go utils.CheckCountry([]string{"Ukraine"})
+	countries := strings.Split(*countryList, ",")
+	if !utils.CheckCountry(countries) && *strictCountryCheck {
```

Comment:
@Arriven updated. The only reason I didn't do it this way is because `countryList` has default option and _technically_ this changes behavior which I wasn't sure if something you want to do.

Reply:
@Arriven How about https://github.com/Arriven/db1000n/pull/302/commits/6a46150402aee95e61b7cbef113940fde28d2736 ?

Reply:
I just realized you might be asking something else.

Do you mean that if `strictCountryCheck` is enabled, this code should be blocking but if it's not enabled, it should be called as go routing?

Reply:
no, I meant basically what you've done but also entering that code if just `strictCountryCheck` i set to true (i.e. you don't want to block specific countries but still want to fail the executable if no connection can be made). But I'm not sure that a valid use case so if you've done it this way it's completely fine for me